### DISCUSSION TITLE
Farewell 2021, Welcome 2022: a personal post · poolp.org

### DISCUSSION THREAD

**Discussion by poolpOrg** [poolpOrg/poolp.org#147] [Articles]
TL;DR: A retrospective of 2021

**Comment by AdrienFromToulouse** [comment]
Hey @poolpOrg ! in regard to drawing you may give a try to sketch noting, also called "facilitation graphique".
I would recommend some training from my friend https://www.linkedin.com/in/cyrille-francois-80354618/
A lot to dig in on that topic. Some techniques are very close to how you would write a "BD".
I may find some interesting stuffs on utube too.

Cheers!

**Reply by poolpOrg** [reply]
Yeah, this whole thing came from me hitting this channel by accident:

https://www.youtube.com/channel/UCLAFr1dkQ7OF0b4rnAHkpqw

I've bought the training pack which is well done, full of videos and explanations.

**Comment by wesley974** [comment]
Bonne & heureuse année Gilles!

**Reply by poolpOrg** [reply]
Merci, également 🍾
